### ISSUE TITLE
Media copyright icons set

### ISSUE THREAD

**Issue by iamrishishah** [Remix-Design/RemixIcon#676]
## Icon Request

* Icon name: image copyright, video copyright, audio copyright, content copyright
* Screenshots of similar icons:
<img width="1131" alt="image" src="https://github.com/Remix-Design/RemixIcon/assets/49694573/533c3f51-5a5c-47e5-8d1b-fba5eedc0398">

**Comment by xiaochunjimmy** [comment]
Hi @iamrishishah , I think we've got the copyright icons in our library.
https://remixicon.com/icon/copyright-line

You mean an copyright symbol in an image or an audio icon?

**Comment by iamrishishah** [comment]
Yes, but I require a very particular one. Envision a video, audio, content, and photo with a copyright icon. Thus, consider that you are providing your users with this as a service. Your services must be displayed as icons.

**Comment by xiaochunjimmy** [comment]
Currently, there have been many requests for other icons accumulated. Based on my initial judgment, I may not prioritize this icon very highly.
